modules/vebal/vebal.gql

```diff
@@ -1,5 +1,6 @@
 extend type Query {
     veBalGetUserBalance(chain: GqlChain, address: String): AmountHumanReadable!
+    veBalGetUserBalances(address: String): [GqlVeBalBalance!]!
```

Comment:
Wouldnt it be better to change `veBalGetUser` to take a chains array? that's more aligned with all other queries

Reply:
Added chains array to the balances